### ISSUE TITLE
Titlebar size & Titlebar subview bug

### ISSUE THREAD

**Issue by GoLdO** [reapp/reapp#66]
Hello, 

I'm just new to reapp (and congrats btw!), i downloaded all the latest versions of everything and followed the tutorial : 
http://reapp.io/docs-builds.html
Juste launched it on xcode and got this : 

![ios simulator screen shot 13 mai 2015 15 46 18](https://cloud.githubusercontent.com/assets/518930/7612159/68f907c8-f989-11e4-8138-35a7a122458c.png)

The title bar seems so small, I guess it can be fixed by fixing a value ? but the main problem is when i click on the subview button, the titlebar seems broken : 

![ios simulator screen shot 13 mai 2015 15 46 21](https://cloud.githubusercontent.com/assets/518930/7612160/68ff32f6-f989-11e4-9b61-66d48a43e218.png)

And the other hand, when i run 'reapp run' and check it on chrome browser, i have no issue.

Thanks

**Comment by morenoh149** [comment]
What browsers have you tried? I only see this on chrome stable with extensions (chrome private browsing it goes away)

**Comment by GoLdO** [comment]
The Cordova simulator uses safari i guess ? On my latest chrome stable x64 web browser it works fine (with extensions) 
Only tried this, sorry.

**Comment by morenoh149** [comment]
@GoLdO no if you've loaded the app in the simulator it's a native app. Unless you opened safari in the simulator and navigated to your app (doubtful).

**Comment by GoLdO** [comment]
Yes sorry, it's getting late, im tired :x And Yes, i tried to run the app in safari web browser too :D It did work when i built for web, and hosted it on my mac.

**Comment by morenoh149** [comment]
my app looks fine on safari Yosemite

**Comment by GoLdO** [comment]
Yes, but not in a vanilla native app.. like you said in https://github.com/reapp/reapp/issues/53, it doesn't seem to be normal, does it ?

**Comment by morenoh149** [comment]
Indeed. Was just providing another data point.

**Comment by gorillatron** [comment]
I have this same bug when I open the app in a fullscreen "added to homescreen" app on my iphone. No visual bug when opening it in regular safari on same iphone.

**Comment by marcelalburg** [comment]
I've the same problem in cordova 5.1.1 and on a real iPhone (in my cas 5s) the visual error occurs too.
